web-portal/pages/admin-venue-edit/index.vue

```diff
@@ -0,0 +1,139 @@
+import { mapGetters, mapActions } from 'vuex'
+
```

Comment:
This probably shouldn't be here, outside the `script` block.

Reply:
fixed